src/Components/LandingPage.js

```diff
@@ -4,23 +4,58 @@ import styled from 'styled-components';
 
 const LandingPage = () => {
     return (
-        <section className="landing-page">
-            <img className="landing-page-img" src="" alt="crafts-in-action"/>
-            <div className="search-and-about-container">
-                <h2>About</h2>
-                <p>about me goes here</p>
+        <LandingPageSection className="landing-page">
+            <LandingImage className="landing-page-img" src={require ("../assets/bannerImage.png")} alt="crafts-in-action"/>
+            <Contents className="search-and-about-container">
+                <About>
+                    <AboutH2>About</AboutH2>
+                    <AboutP>Craft Circle is an online community where crafters can borrow and donate supplies in an effort to make creative outlets more accessible. Currently, this platform is meant for community members who reside in the Denver Metro area.</AboutP>
+                </About>
                 <form className="search-bar">
                     <label>Search</label>
                     <input type="text"></input>
                     <button className="search-button">search</button>
                 </form>
-            </div>
-        </section>
+            </Contents>
+        </LandingPageSection>
     )
 }
 
 export default LandingPage
 
+const LandingImage = styled.img`
+width: 80vw;
+border-radius: 25px;
+box-shadow: 40px 40px rgba(239,203, 199, 1);
+margin: 10px;
+`
+const AboutH2 = styled.h2`
+color: #AD92C0;
```

Comment:
Future refactor note: We'll need to come back and edit these colors to be more reusable using variables